src/cmd/describegpt.rs

```diff
@@ -0,0 +1,118 @@
+static USAGE: &str = r#"
+Infers extended metadata about a CSV using summary statistics.
+
+Note that this command uses a LLM for inference and is therefore prone to inaccurate
+information being produced. Ensure verification of output results before using them.
+
+For examples, see https://github.com/jqnatividad/qsv/blob/master/tests/test_describegpt.rs.
+
+Usage:
+    qsv describegpt [options] [<input>]
+    qsv describegpt --help
+
+describegpt options:
+    -A, --all              Print all extended metadata options output.
+    --description          Print a general description of the dataset.
+    --dictionary           For each field, prints an inferred type, a 
+                           human-readable label, and a description.
+    --tags                 Prints tags that categorize the dataset. Useful
+                           for grouping datasets and filtering.
```

Comment:
For tags, let's keep it simple at the initial MVP to just use summary statistics.

But we can possibly also leverage the `qsv frequency` command to provide the LLM a frequency table of the top 20 values of each field for the LLM to better infer the category of the dataset.

We can wait until the next release to use frequency tables @rzmk , but I just wanted to put it on your radar screen.